manifests/init.pp

```diff
@@ -64,6 +64,20 @@
 # $package_names::      Packages that this module ensures are present instead of the default
 #                       type:Array[String]
 #
+# $manage_repo::        Whether to manage the yum repository
+#                       type:Boolean
+#
+# $repo::               Which yum repository to install. For example
```

Comment:
I originally copied this from puppet-foreman into puppet-candlepin. We could change it to ```repo_version``` and similar in puppet-candlepin.